composer.json

```diff
@@ -8,39 +8,40 @@
         "AFL-3.0"
     ],
     "require": {
-        "magento/product-community-edition": "2.1.7",
         "composer/composer": "@alpha",
         "magento/module-bundle-sample-data": "100.1.*",
-        "magento/module-theme-sample-data": "100.1.*",
+        "magento/module-catalog-rule-sample-data": "100.1.*",
         "magento/module-catalog-sample-data": "100.1.*",
-        "magento/module-tax-sample-data": "100.1.*",
-        "magento/module-customer-sample-data": "100.1.*",
         "magento/module-cms-sample-data": "100.1.*",
-        "magento/module-widget-sample-data": "100.1.*",
-        "magento/module-catalog-rule-sample-data": "100.1.*",
-        "magento/module-sales-rule-sample-data": "100.1.*",
-        "magento/module-sales-sample-data": "100.1.*",
-        "magento/module-grouped-product-sample-data": "100.1.*",
+        "magento/module-configurable-sample-data": "100.1.*",
+        "magento/module-customer-sample-data": "100.1.*",
         "magento/module-downloadable-sample-data": "100.1.*",
+        "magento/module-grouped-product-sample-data": "100.1.*",
         "magento/module-msrp-sample-data": "100.1.*",
-        "magento/module-configurable-sample-data": "100.1.*",
+        "magento/module-offline-shipping-sample-data": "100.1.*",
         "magento/module-product-links-sample-data": "100.1.*",
-        "magento/module-wishlist-sample-data": "100.1.*",
         "magento/module-review-sample-data": "100.1.*",
+        "magento/module-sales-rule-sample-data": "100.1.*",
+        "magento/module-sales-sample-data": "100.1.*",
         "magento/module-swatches-sample-data": "100.1.*",
-        "magento/sample-data-media": "100.1.*",
-        "magento/module-offline-shipping-sample-data": "100.1.*"
+        "magento/module-tax-sample-data": "100.1.*",
+        "magento/module-theme-sample-data": "100.1.*",
+        "magento/module-widget-sample-data": "100.1.*",
+        "magento/module-wishlist-sample-data": "100.1.*",
+        "magento/product-community-edition": "2.1.7",
+        "magento/sample-data-media": "100.1.*"
     },
     "require-dev": {
-        "phpunit/phpunit": "4.1.0",
-        "squizlabs/php_codesniffer": "1.5.3",
-        "phpmd/phpmd": "@stable",
-        "pdepend/pdepend": "2.4.0",
         "fabpot/php-cs-fixer": "~1.2",
         "lusitanian/oauth": "~0.3 <=0.7.0",
-        "sebastian/phpcpd": "2.0.0"
+        "pdepend/pdepend": "2.4.0",
+        "phpmd/phpmd": "@stable",
+        "phpunit/phpunit": "4.1.0",
+        "sebastian/phpcpd": "2.0.0",
+        "squizlabs/php_codesniffer": "1.5.3"
     },
     "config": {
+        "sort-packages": true,
```

Comment:
Added this, then ran

```
$ composer require magento/product-community-edition:2.1.7 && composer require --dev phpunit/phpunit:4.1.0
```

to trigger sorting of packages, but without actually updating any of them.